portfolio/src/main/java/com/google/sps/servlets/RandomMessageServlet.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps.servlets;
+
+// import com.google.sps.data.ServerStats;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/random-message")
+
+public class RandomMessageServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        ArrayList<String> messages = new ArrayList<String>();
+        messages.add("first message");
+        messages.add("second message");
+        messages.add("third message");
+        // Convert messages to JSON
+        String json = convertToJson(messages);
+
+        // Send the JSON as the response
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+    private String convertToJson(ArrayList<String> messages) {
```

Comment:
It'd be better to use gson to create the JSON string from the messages array than building it yourself.